src/styles.scss

```diff
@@ -73,3 +76,54 @@ body {
   height: 100%;
   margin: 0;
 }
+
+:root {
+  .alertify-notifier {
+    display: flex;
+    width: 436px;
+    flex-direction: column;
+    align-items: flex-end;
+
+    &.ajs-top {
+      top: 92px;
+    }
+
+    &.ajs-right {
+      right: 64px;
+
+      .ajs-message.ajs-visible {
+        right: 0;
+      }
+    }
+
+    .ajs-message {
+      width: auto;
+      max-width: 436px;
+      padding: 2em;
+      margin-bottom: 1em;
+      background-color: map-get($sormas-primary, 50);
+      border-radius: 5px;
+      box-shadow: 0 4px 5px rgba(0, 0, 0, 0.14), 0 1px 10px rgba(0, 0, 0, 0.12),
```

Comment:
Please add a  variable in variables.scss and use it here like : rgba($black, 0.14);

Reply:
i've used new way of adding shadows